.github/workflows/percy-workflow.yml

```diff
@@ -4,13 +4,6 @@ name: Percy
 # configure each event separately. You must append a colon (:) to all events,
 # including events without configuration.
 on:
-  push:
```

Comment:
So we wouldn't run percy on push to master? I guess we never really push straight to master... So there's no need to check how the site looks, the push wouldn't be blocked. But incase there is a push to master, couldn't percy notify us if the pages don't look right?

Reply:
It certainly could, though at the moment we're not doing a good job of seeing those kinds of alerts. But that seems like a good enough reason to leave it in for now. I'll update the PR!